cfgov/jinja2/v1/_includes/molecules/pagination.html

```diff
@@ -1,29 +1,31 @@
 
 {# ==========================================================================
 
-   pagination()
+   Pagination
 
    ==========================================================================
 
    Description:
 
    Builds pagination markup when given:
 
-   posts.page:          Current page number of the iterable list.
+   posts:                        TODO: fill in data type and description.
 
-   posts.filters:               Current filters applied to the iterable list.
+   posts.paginator:              TODO: fill in data type and description.
 
-   posts.next_button_href:      Link to the next page.
+   posts.paginator.num_pages:    Number of total pages.
 
-   posts.pages:                 Number of pages in the iterable list.
+   posts.number:                 TODO: fill in data type and description.
 
-   posts.previous_button_href:  Link to the previous page.
+   posts.previous_page_number(): Method to return previous page number.
+
+   posts.page:                 Current page number of the iterable list.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2032/commits/e95d3aa47240421b21ff48e05041ebf63b1cf6b5